libraries/helpers_network.rb

```diff
@@ -92,7 +92,6 @@ def consolidate_ipam(subnets, ranges, gateways, auxaddrs)
             data[s]['Gateway'] = g
             match = true
           end
-          raise "no matching subnet for gateway #{s}" unless match
```

Comment:
is there a gateway being set? still confused on how this is being triggered if `gateways` is empty.